src/components/Modals/CreateEventModal.jsx

```diff
@@ -73,6 +88,39 @@ export class CreateEventModal extends React.Component {
     this.endTimeStamp.setMinutes(date.getMinutes());
   }
 
+  disabledProgressCircle() {
+    this.setState( {isLoading: false} );
+  }
+
+  openLogInBox () {
+    const { onRequestClose } = this.props;
+    onRequestClose();
+  }
+
+  renderProgressCircle() {
+    if(this.props.userId){
+      if(this.state.isLoading) {
+        return ( 
+          <div>
+            <CircularProgress />
+          </div> 
+        );
+      }
+      return (
+        <div>
+          <button type='submit' className="create-btn" onClick={() => { this.setState({ isLoading: true }); this.addNewEvent(); }}>CREATE</button>
+        </div>
+      );   
+    }
+    else {
+      return (
+        <div>
+          <button type='submit' className="create-btn" onClick={() => { this.openLogInBox(); }}>LOG IN</button>
```

Comment:
You can remove the openLogInBox function and just have a one-liner here `onClick={() => this.props.onRequestClose() }`